models/demos/ttnn_resnet/tests/test_perf_ttnn_resnet.py

```diff
@@ -311,14 +320,17 @@ def run_perf_resnet(
     )
     torch_resnet50.to(torch.bfloat16)
 
-    tt_resnet50 = resnet50(
-        device=device,
-        parameters=parameters,
-        batch_size=batch_size,
-        model_config=model_config,
+    test_infra = create_test_infra(
+        device,
+        batch_size,
+        model_config["ACTIVATIONS_DTYPE"],
+        model_config["WEIGHTS_DTYPE"],
+        model_config["MATH_FIDELITY"],
         dealloc_input=True,
-        final_output_mem_config=ttnn.DRAM_MEMORY_CONFIG if "trace" in model_version else ttnn.L1_MEMORY_CONFIG,
+        final_output_mem_config=ttnn.DRAM_MEMORY_CONFIG,
+        model_location_generator=model_location_generator,
```

Comment:
Can we avoid using test_infra in the perf test? I'd prefer to invoke the actual model in the perf test if possible to avoid any potential overhead someone might add to test_infra but this isn't  really a blocker.

This change seems to have hardcoded final output to DRAM, whereas previously it was only DRAM if running trace configuration. Was there a reason this was changed?

Reply:
oops, will change back